src/cargo/core/features.rs

```diff
@@ -330,6 +330,7 @@ pub struct CliUnstable {
     pub avoid_dev_deps: bool,
     pub minimal_versions: bool,
     pub package_features: bool,
+    pub package_features2: bool,
```

Comment:
Are you worried about breakage if we change the existing unstable feature `package_features`? I'd be ok with simply updating that since we're more likely to stabilize this than that.

Reply:
I'm fine with merging them (will do that soonish). I wanted to get feedback first, and keeping them separate was a way to highlight how they differ.